### ISSUE TITLE
Success percentage shown for tests are not correct.

### ISSUE THREAD

**Issue by pasindujw** [wso2/testgrid#953]
**Description:**
Currently we are calculating success percentage as;

`this.successPercentage = (double) totalSuccess / ((double) totalSuccess + (double) totalFail) * 100d;
`
AFAIU We should consider total tests run for the denominator.

**Suggested Labels:**
Bug

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by kasunbg** [comment]
@pasindujw yes, true. I guess we did it this way because there skipped tests as well. We need to capture skipped tests across the board in testgrid. So, let's keep this in mind, and address this when we get to introducing support for skipped tests.
